pyproject.toml

```diff
@@ -1,3 +1,32 @@
+[tool.poetry]
+name = "pandas-stubs"
+version = "0.0.1"
+description = "Stubs for pandas"
+authors = ["Breno Fernandes <brenojesusfernandes@pm.com>"]
+packages = [
+    { "include" = "pandas-stubs"}
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
```

Comment:
maybe we should say <=3.10 since we only support pandas up to that version

Reply:
Nice, I did not even notice. I'm gonna fix this. Noted.